projectPage.html

```diff
@@ -6,32 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="HandheldFriendly" content="true">
-
     <title>PROJECTS</title>
 </head>
 
-<div class="all">
+<!-- <div class="all"> -->
```

Comment:
can't have div containing a body. body is the highest tier element, so you can have a div inside the body. but you cant have the body inside of a div.